legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -103,19 +103,30 @@ version (documented on this page) and the "legacy" version (documented at
 
 {% h4 Optional Fields %}
 
-|     Field      |  Type  |                                         Description                                          |
-| -------------- | ------ | -------------------------------------------------------------------------------------------- |
-| bitcoinAddress | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                              |
-| cennzAddress   | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                |
-| wavesAddress   | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                         |
-| productCodes   | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`        |
-| collectionIds  | Array  | Supported collectionIds for the payment request, when the "assetType" is `centrapay.token.*` |
+|     Field            |  Type  |                                         Description                                                                 |
+| -------------------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
+| bitcoinAddress       | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                                                     |
+| cennzAddress         | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                                       |
+| wavesAddress         | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                                                |
+| productCodes         | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`                               |
+| acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|
 
 ★  For payment options which specify an address, there's a requirement to make a transaction on an external ledger.
 Once you have made that payment, you can use the transaction id to [Pay a Payment Request](#pay) using the legacy payment API.
 
 <a name="payment-condition">
 
+### Accepted Collection
+
+Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] is accepted for a particular Payment Request, and what line items the token can be used to purchase.
```

Comment:
Will Accepted Collections be returned on the Payment Option or the Payment Request? The first sentence reads as if the Accepted Collections is returned on the Payment Request.

Reply:
It will be returned on the Payment Option, I will update it to be more clear. :)

Reply:
I have updated this, take a look and let me know what you think. :)